RFCs/FS-1033-Deprecate-places-where-seq-can-be-omitted.md

```diff
@@ -0,0 +1,313 @@
+# F# RFC FS-1033 - Deprecate places where seq can be omitted.
+
+The design suggestion [Deprecate places where seq can be omitted](https://github.com/fsharp/fslang-suggestions/issues/1033) has been marked "approved in principle."
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Deprecate places where seq can be omitted](https://github.com/fsharp/fslang-suggestions/issues/1033)
+- [x] Approved in principle
+- [x] [Implementation](https://github.com/dotnet/fsharp/pull/17772)
+- [x] [Discussion](https://github.com/fsharp/fslang-design/discussions/788)
+
+# Summary
+
+Deprecate places where seq can be omitted.
+
+# Motivation
+
+- Is confusing that we don't need to use `seq` in `{ start..finish }` but you need to do `seq` in `{ x; y }` e.g.
+
+```fsharp
+  { 1..10 } // No error
+  { 1..3..10 } // No error
+  { 1;10 } // Error: Invalid record, sequence or computation expression. Sequence expressions should be of the form 'seq { ... }'
+```
+
+- Having only one way to create a sequence will make the language more consistent and easier to understand.
+
+- Prevents the accidental override of index range operator.
+```fsharp
+let (..) _ _ = "Im an index range operator"
+
+let x = { 1..10 } // Here the index range operator is being overridden and `x` will be a string instead of a sequence.
+```
+
```

Comment:
``suggestion
- The FSharp-Spec-4.1 already mentions that the ommission of the `seq` keyword is not recommended and that it might be deprecated (§6.3.12).

```